packages/d3fc-webgl/src/buffer/adjacentElementAttribute.js

```diff
@@ -55,12 +55,21 @@ export default (minOffset = 0, maxOffset = 0) => {
         };
 
         rebind(offsetAttribute, base, 'location');
+        rebind(offsetAttribute, projector, 'dirty');
```

Comment:
As discussed, please can we go with `clear()` rather than `dirty(true)` so as to prevent users from calling `dirty(false)`